package.json

```diff
@@ -35,39 +35,47 @@
     "src/*.h",
     "drafter/common.gypi",
     "drafter/drafter.gyp",
-    "drafter/src/*.cc",
-    "drafter/src/*.h",
-    "drafter/src/backend/*.cc",
-    "drafter/src/backend/*.h",
-    "drafter/src/parser/*.cc",
-    "drafter/src/parser/*.h",
-    "drafter/src/utils/*.cc",
-    "drafter/src/utils/*.h",
-    "drafter/src/utils/so/*.cc",
-    "drafter/src/utils/so/*.h",
-    "drafter/src/utils/log/*.cc",
-    "drafter/src/utils/log/*.h",
-    "drafter/src/refract/*.cc",
-    "drafter/src/refract/*.h",
-    "drafter/src/refract/dsd/*.cc",
-    "drafter/src/refract/dsd/*.h",
-    "drafter/ext/boost/boost",
-    "drafter/ext/PEGTL/include/tao/*.hpp",
-    "drafter/ext/PEGTL/include/tao/pegtl/*.hpp",
-    "drafter/ext/PEGTL/include/tao/pegtl/analysis/*.hpp",
-    "drafter/ext/PEGTL/include/tao/pegtl/contrib/*.hpp",
-    "drafter/ext/PEGTL/include/tao/pegtl/internal/*.hpp",
-    "drafter/ext/snowcrash/src/*.cc",
-    "drafter/ext/snowcrash/src/*.h",
-    "drafter/ext/snowcrash/src/posix/RegexMatch.cc",
-    "drafter/ext/snowcrash/src/win/RegexMatch.cc",
-    "drafter/ext/snowcrash/ext/markdown-parser/src/*.cc",
-    "drafter/ext/snowcrash/ext/markdown-parser/src/*.h",
-    "drafter/ext/snowcrash/ext/markdown-parser/ext/sundown/src/*.c",
-    "drafter/ext/snowcrash/ext/markdown-parser/ext/sundown/src/*.h",
-    "drafter/ext/snowcrash/ext/markdown-parser/ext/sundown/html/*.c",
-    "drafter/ext/snowcrash/ext/markdown-parser/ext/sundown/html/*.h",
-    "drafter/ext/variant/include/mpark/*.hpp"
+    "drafter/packages/drafter/src/*.cc",
```

Comment:
Drafter sources have been significantly altered in https://github.com/apiaryio/drafter/pull/782, updating the submodule means these file lists have to be updated.

The smoke tests show that a packaged npm package is build-able, this shows that it contains all the needed files (most importantly, it proves that this list is correct).